Library/Homebrew/style.rb

```diff
@@ -298,7 +298,7 @@ def self.json_result!(result)
 
     def self.shell_scripts
       [
```

Comment:
Might be worth doing `.map(&:realpath)` on this entire array to be sure/future-proof.